exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -275,6 +275,16 @@ func (e *exporter) sendBatch(ctx context.Context, lines []string) error {
 		return consumererror.NewPermanent(fmt.Errorf("metrics ingest v2 module not found - ensure module is enabled and endpoint is correct"))
 	}
 
+	if resp.StatusCode == http.StatusTooManyRequests {
+		return consumererror.NewPermanent(
+			fmt.Errorf("The server responded that too many requests have been sent. Please check your export interval and batch sizes, or see the following for more information: https://www.dynatrace.com/support/help/dynatrace-api/basics/access-limit"),
+		)
+	}
+
+	if resp.StatusCode >= http.StatusBadRequest {
+		return consumererror.NewPermanent(fmt.Errorf(`Received error response status: "%v"`, resp.Status))
```

Comment:
This just prints the status code right? Maybe: 
```suggestion
		return consumererror.NewPermanent(fmt.Errorf(`Received error response status code: "%v"`, resp.Status))
Write
```

Reply:
This actually prints the status code and status message, e.g. "200 OK". You can see an example in [this assertion](https://github.com/dynatrace-oss-contrib/opentelemetry-collector-contrib/blob/40e819347efddb4f52536ce665d6c1774248d3e8/exporter/dynatraceexporter/metrics_exporter_test.go#L582).

Reply:
Ah perfect, ignore my comment then!